src/ui/screens/online_matchmaking_screen.py

```
@@ -0,0 +1,320 @@
+        except Exception as exc:
+            self.status_message = "Failed to join matchmaking"
+            self.status_color = RED
+            self.error_detail = str(exc)
+
+    def _leave_queue(self) -> None:
+        if not self._in_queue or self._left_queue:
+            return
+        try:
+            self.api_client.matchmaking_leave()
+        except Exception:
+            pass
```

Ruff (0.14.6)

`try`-`except`-`pass` detected, consider logging the exception

(S110)